api/controllers/InvitationController.js

```diff
@@ -125,5 +125,26 @@ module.exports = {
         })
       })
     .then(results => res.ok({results}))
+  },
+
+  reInviteAll: function (req, res) {
+    const communityId = res.locals.community.id
+    return Invitation.where({community_id: communityId, used_by_id: null})
```

Comment:
you also want to filter out invitations where used_by_id is null but there is someone with a matching email address, as we do in `InvitationController.find`